.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -56,9 +56,9 @@ _Will this PR introduce a change that impacts Forem members or creators, the
 development process, or any of our internal teams? If so, please note how you
 will share this change with the people who need to know about it._
 
-- [ ] I've updated the [Developer Docs](https://developers.forem.com) and/or
-      [Admin Guide](https://admin.forem.com/), or
+- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
       [Storybook](https://storybook.forem.com/) (for Crayons components)
+- [ ] This PR changes the Admin area and I have fill out the [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md) issue template so Community Success can update the Admin Docs. 
```

Comment:
_Quick suggestion:_
```suggestion
- [ ] This PR changes the Admin area and I have filled out the [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md) issue template so Community Success can update the Admin Docs. 
```

Reply:
Michael made this suggestion:
“This PR changes the Forem platform and our documentation needs updated. I have filled out the [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md) issue template so Community Success can help update the Admin Docs appropriately.” since we will potentially need to update Admin docs even if the admin area isn't worked on. So I am going to change the language to that instead. Does that sound good to you @juliannatetreault ?

Reply:
Either of the two sound good to me! We could even potentially do something like this as well: 

> This PR changes the Admin area and I have filled out the [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md) issue template so Community Success can update the Admin Docs if necessary.

I'm good with whatever you decide on, though @cmgorton. 😊 My only suggestion to the copy that you just suggested in your most recent comment is that we change:
>...and our documentation needs updated.
to read:
> ...and our documentation needs to be updated.

Reply:
Sounds good Julianna. Thanks 😄